zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -5268,6 +5268,51 @@ components:
             As a result, the completion request is rejected and the task remains active.
             Defaults to false.
           nullable: true
+        corrections:
+          $ref: "#/components/schemas/JobResultCorrections"
+        correctedAttributes:
+          type: array
+          description: List of the attributes that were corrected by the worker.
+          items:
+            type: string
+          nullable: true
```

Comment:
As noted in https://github.com/camunda/camunda/pull/25654/files#r1879850809, exposing `correctedAttributes` in the REST API may not be necessary.